client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -71,52 +126,75 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 			reduce(
 				this.fields,
 				( eventProps, value, field ) => {
-					const changed = get( settings, [ 'businessAddress', field ] ) !== this.state[ field ];
+					const changed =
+						get( settings, [ 'businessAddress', field ] ) !== this.state.fields[ field ];
 					eventProps[ `${ field }_changed` ] = changed;
 					return eventProps;
 				},
 				{}
 			)
 		);
 
-		this.props.saveJpoSettings( siteId, { businessAddress: this.state } );
-
-		page( this.props.getForwardUrl() );
+		this.props.saveJpoSettings( siteId, { businessAddress: this.state.fields } );
 	};
 
 	hasEmptyFields = () => {
-		return some( omit( this.state, 'state' ), val => val === '' );
+		return some( omit( this.state.fields, 'state' ), val => val === '' );
 	};
 
-	render() {
-		const { basePath, isRequestingSettings, translate } = this.props;
-		const headerText = translate( 'Add a business address.' );
-		const subHeaderText = (
+	renderHeader() {
+		const { translate } = this.props;
+		const headerText = translate( 'Help your customers find you with Jetpack.' );
+		const subHeaderText = translate(
+			"Add your business address and a map of your location with Jetpack's business address widget. " +
+				"You can adjust the widget's location later."
+		);
+
+		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+	}
+
+	renderActionTile() {
+		const { isConnected, siteUrl, translate } = this.props;
+
+		const connectUrl = addQueryArgs(
+			{
+				url: siteUrl,
+				// TODO: add a parameter to the JPC to redirect back to this step after completion
+				// and in the redirect URL include the ?action=add_business_address parameter
+				// to actually trigger the form display action after getting back to JPO
+			},
+			'/jetpack/connect'
+		);
+		const href = ! isConnected ? connectUrl : null;
+
+		return (
 			<Fragment>
-				{ translate(
-					'Enter your business address to add a widget containing your address to your website.'
-				) }
-				<br />
-				{ translate(
-					'You can add a map based on this information and change where this widget is located later on.'
-				) }
+				{ this.renderHeader() }
+
+				<TileGrid>
+					<Tile
+						buttonLabel={ translate( 'Add a business address' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Add a business address.' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jpo-ba-step-jpc-intro-and-success)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).